src/main.rs

```diff
@@ -44,5 +53,17 @@ fn main() {
                 std::process::exit(1)
             }
         })
+        .runcommand(match matches.value_of("run").unwrap() {
```

Comment:
I think it would make more sense to keep omit `default_value` and pass `matches.value_of` as a whole to the `runcommand` function and have the logic be there to see if the value is an `Option::Some`, etc.

Reply:
That makes sense, thanks.

Reply:
@dariusc93, I'll let this thing hang for a while, gonna `amend && push -f` later tomorrow.